tools/kube-burner/template/application.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: app-{{.Iteration}}
+  namespace: <argocd-namespace>
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/Mangaal/argocd-performance-test
```

Comment:
Should this be updated to the argoproj-labs repo eventually?

Reply:
Yes, I need to update this